src/main/java/com/actor/example/MultiplierActor.java

```diff
@@ -19,6 +19,12 @@ public int getMultiplier() {
         }
     }
 
+    private void TestMethod() {
+        List<Integer> list = new ArrayList<>();
+        list.add(1);
+        System.out.prinlln(list);
+        }
+
     @Override
     public Receive createReceive() {
```

Comment:
There is a typo in the method. "System.out.prinlln" should be "System.out.println".